modules/admin_manual/pages/release_notes.adoc

```diff
@@ -34,6 +35,119 @@
+
+==== Feedback
+As mentioned, features are still being added to ownCloud Web and the new web interface can't yet cover every use case of the Classic interface. To further shape the new product and to determine the development priorities it is of utmost importance to consider user feedback. We're very grateful for any hints or feedback that you supply via the following channels
+
+- Survey: LINK
+- GitHub: https://github.com/owncloud/web[owncloud/web]
+- Chat: talk.owncloud.com / #web
+
+With one of the next ownCloud Web releases, there will be an integrated feedback feature that will allow users to directly report about their experience.
+
+=== Improved Usability for "Add to ownCloud" on Public Links
+
+The usability of the "Add to ownCloud" feature on public link pages has been improved (https://github.com/owncloud/core/pull/38712[#38712]). Research on the feature has shown that most users use the feature to add links to the same ownCloud installation they originate from instead of creating federated shares with other ownCloud installations. For this reason there is a new default behavior which allows to add a link to a user account on the ownCloud installation the public link originates from with just a click and without requiring any further input. Advanced users can use a dropdown menu to add a link to a user account on another ownCloud installation.
+
+=== Modernized Login Page
+
+The login page has received several improvements in regards of design and usability (https://github.com/owncloud/core/pull/38506[#38506]). A login button and labels for the input fields have been added and the "Alternative login" buttons (e.g., for authentication with OpenID Connect) have been redesigned to give a more modern look and feel. In this context, also the ownCloud Enterprise Theme has received improvements that are shipped with Server 10.8.
+
+=== Improvements for External Storages
+
+* Performance and caching improvements for external storages (e.g., Windows Network Drives) have been added https://github.com/owncloud/core/pull/38804[#38804]
+* Renaming or deleting a Windows Network Drive mount point from the files list is now properly prevented https://github.com/owncloud/core/pull/38709[#38709] https://github.com/owncloud/core/pull/38819[#38819]
+* Passwords in the external storage configuration can now be stored encoded in the database (e.g., for Windows Network Drive mount points). With the next release of the Windows Network Drive app, these can also be used to set up the WND Listener. https://github.com/owncloud/core/pull/38728[#38728]
```

Comment:
`configuration can now be stored` or `configuration is now stored` ?

Reply:
"can now", as it depends on the external storage apps like WND to actually do it.